core/src/main/java/org/mobilitydata/gtfsvalidator/model/ValidationReportDeserializer.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
I would have expected this class to be in an `.io` package, not `model`, because it handles I/O, and isn't a container class for data.